README.md

```diff
@@ -102,6 +102,49 @@ Other options are detailed in the [AspectJ Ant tasks][] documentation.
 [AjcTask]: http://www.eclipse.org/aspectj/doc/next/devguide/antTasks-iajc.html
 [AspectJ Ant tasks]: http://www.eclipse.org/aspectj/doc/next/devguide/antTasks.html
 
+#### Using Gradle
+
+[gradle example]: https://github.com/paegun/metrics-aspectj-gradle-example
+A working [gradle example][] is available, but each integration point is described here.
+
+##### build.gradle snippets
+
+```java
```

Comment:
A brief scour of gradle+markdown documents yielded `groovy` as the appropriate language.